### ISSUE TITLE
Wrong check in Jupyter Notebook for lesson Keras

### ISSUE THREAD

**Issue by bartmate** [udacity/sdc-issue-reports#152]
Hi,
in several places of the notebook it is said that you have to run two epochs. After the execution, the check-part (which is not allowed to be changed) checks for the result of the 1st epoch instead of the result of the 2nd epoch (see the 2nd index of history.history[][]). The result is that the notebook will not accept a good solution.

What we have:
assert(history.history['val_acc'][0] > 0.9), "The validation accuracy is: %.3f" % history.history['val_acc'][0]

What should be there:
assert(history.history['val_acc'][1] > 0.9), "The validation accuracy is: %.3f" % history.history['val_acc'][1]

In some cases it is not said that you should have 2 epochs, in this case I think the last epoch should be checked (not the first one): history.history['val_acc'][ index_of_last_epoch ] > 0.9.

----------------------------------------------

Related problem: in the 'Train the Network' chapter in the assert statement there is a missing index at the end: history.history['val_acc'] instead of history.history['val_acc'][ XXX ].  

Regards,
  BartMate

**Comment by domluna** [comment]
@bartmate in the latest notebook both these issues been fixed. The history assertion is for the last history value now.
